moja-kuhinjica-frontend/src/components/modal/signUp/SignUpModal.tsx

```diff
@@ -1,54 +1,83 @@
-import React from 'react'
+import React, { use, useState } from 'react'
 import Modal from 'react-modal'
-import Image from 'next/image'
-import { useForm } from 'react-hook-form'
+import { useForm, FieldValues } from 'react-hook-form'
 import { FormInput } from '@/components/input/FormInput'
 import styles from './SignUpModal.module.scss'
 import { bgModal } from '@/constants/constants'
 import profile from '../../../../public/static/assets/images/profile.svg'
 import email from '../../../../public/static/assets/images/email.svg'
 import password from '../../../../public/static/assets/images/password.svg'
 import mobile from '../../../../public/static/assets/images/mobile.svg'
-import google from '../../../../public/static/assets/images/google.svg'
-import mailSignUp from '../../../../public/static/assets/images/mailSignUp.svg'
+import UserService from '@/service/User.service'
+import { ErrorLabel } from '@/components/label/ErrorLabel'
 
 interface ISignUpModalProps {
     modalIsOpen: boolean
-    closeModal: (param: boolean) => void
+    closeModal: () => void
+    openNotificationModal: (email: string) => void
 }
 export const SignUpModal = ({
     modalIsOpen,
     closeModal,
+    openNotificationModal,
 }: ISignUpModalProps): JSX.Element => {
+    const [showError, setShowError] = useState<boolean>(false)
+    const [errorMessage, setErrorMessage] = useState<string>('')
     const {
         register,
         handleSubmit,
         reset,
         formState: { errors },
     } = useForm()
-    const onSubmit = (data: any): void => {
-        reset()
+
+    const signIn = (inputData: FieldValues): void => {
+        console.log(inputData)
+        const res = UserService.signIn(inputData)
+            .then((res) => {
+                closeModal()
+                reset()
+                openNotificationModal(inputData.email)
+            })
+            .catch((err) => {
+                setErrorMessage(err.message)
+                setShowError(true)
+                console.log(err)
+            })
+    }
+
+    const validate = (data: FieldValues): void => {
+        if (data.password === data.confirmPassword) {
```

Comment:
You should use `==` instead of `===`

Reply:
<img width="912" alt="Screenshot 2023-02-27 at 10 29 44" src="https://user-images.githubusercontent.com/73345582/221526137-cac9c129-7c35-411a-ab97-3c34716a27e5.png">